subprojects/GOPATH/src/github.com/microsoft/hivedscheduler/pkg/api/types.go

```diff
@@ -158,3 +158,32 @@ type LazyPreemptionStatus struct {
 	// It was lazy preempted at PreemptionTime.
 	PreemptionTime meta.Time `json:"preemptionTime"`
 }
+
+type VirtualClusterList struct {
+	Items []VirtualCluster `json:"items"`
+}
+
+type VirtualCluster struct {
+	ObjectMeta `json:"metadata"`
+	Status     VirtualClusterStatus `json:"status"`
+}
+
+type VirtualClusterStatus struct {
+	VirtualClusterCapacity *ClusterCapacity `json:"virtualClusterCapacity"`
+}
```

Comment:
Add per gputype status?

Reply:
Maybe we can just expose one or very few most detailed status of our current scheduling, but still make sure that RestServer can easily calculate other status view based on it.

---
In reply to: [368359248](https://github.com/microsoft/pai/pull/4153#discussion_r368359248) [](ancestors = 368359248)